2.train-VAE/scripts/1.optimize_hyperparameters.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import sys
+import pathlib
+import numpy as np
+import pandas as pd
+import hiplot
+from optuna.visualization import plot_param_importances
+import optuna
+import torch
+from torch.utils.data import DataLoader, TensorDataset
+from sklearn.model_selection import train_test_split
+
+from betavae import (
+    BetaVAE,
+    train_vae,
+    evaluate_vae
+)
+from optimize_utils import(
+    get_optimize_args
+)
+
+
+# In[ ]:
+
+
+#Load command line arguments
+args=get_optimize_args()
+
+#Load data
+
+output_dir = pathlib.Path("data")
+
+sys.path.insert(0, "../0.data-download/scripts/")
```

Comment:
consider using [pathlib.Path](https://docs.python.org/3/library/pathlib.html) for all strings that represent paths.

(pathlib has some nifty functionality, and enables cross-OS support; i.e., microsoft uses `/` instead of `\`)

Reply:
Corrected